Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser.py

```diff
@@ -123,8 +137,8 @@ def get_service(self, serviceName, version):
         credentials = AccessTokenCredentials(token, 'Demisto Gmail integration')
         if PROXY or DISABLE_SSL:
```

Comment:
Not sure I am following. I think the code here is ok. Please check. If needed open a separate issue.

Reply:
I'll open a separate issue then.

The problem here is when `DISABLE_SSL` and `PROXY ` are False, and then, the `handle_proxy` method which deletes the proxy env variables is not called and the instance will use the proxy.

Reply:
Ok. Please do a check as from what I've seen it works fine.

Reply:
Checked most calls runs the `get_service` method which handles the proxy.
But the fetch incident command does not do it and so in case `DISABLE_SSL` and `PROXY` are False it uses the proxy configuration.

I'l update the issue here https://github.com/demisto/etc/issues/32155